modules/imagetools/src/main/ts/ephox/imagetools/util/Conversions.ts

```diff
@@ -173,7 +172,7 @@ function blobToDataUri(blob: Blob): Promise<string> {
     const reader = new FileReader();
 
     reader.onloadend = function () {
-      resolve(reader.result);
+      resolve(reader.result as string);
```

Comment:
`FileReader` can return either a `string` or an `ArrayBuffer`, depending on whether you call `readAsDataURL`, `readAsArrayBuffer`, etc... so we have to cast here unfortunately.

Reply:
could a code comment be helpful for this?

Reply:
It's done in quite a number of places and I really didn't want to have to carry that across in every place.